src/components/PlanCard/PlanCard.tsx

```diff
@@ -49,9 +54,11 @@ export const PlanCard = ({
       <PlanNav>
         <PillLabel type="primary"> test label</PillLabel>
         <PillLabel type="secondary"> test label</PillLabel>
+        <PrimaryButton onClick={toggle}>Share</PrimaryButton>
```

Comment:
can you please remove this from the merge, so that we can have just the modal, and it'll be hooked up with the share button in the `PlanCard`